DisplaySettings/DisplaySettings.cpp

```diff
@@ -315,8 +315,14 @@ namespace WPEFramework {
                         LOG_DEVICE_EXCEPTION1(string("HDMI_ARC0"));
                     } 
                     if (portName == "HDMI_ARC0") {
-                        //Set audio port config. ARC will be set up by onTimer()		
-                        if(isPortPersistenceValEnabled &&  m_hdmiCecAudioDeviceDetected) { 
+                        //Set audio port config. ARC will be set up by onTimer()
```

Comment:
Do you want to call InitAudioPorts() on the product where APP is anyway initing to avoid duplication? However you need to test this out...